LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -414,6 +414,7 @@
     "web-platform-tests/subresource-integrity": "import",
     "web-platform-tests/svg": "import",
     "web-platform-tests/svg-aam": "skip",
+    "web-platform-tests/svg/text": "import",
```

Comment:
This is implicit with `    "web-platform-tests/svg": "import",`. Can we undo this change?

Reply:
Will do but will let EWS run so I can do all tweaks in one go. :-)

Reply:
Sounds good!

Reply:
(isn't this autogenerated though?)

Reply:
It is autogenerated. :-)

Reply:
@aprotyas Yeah the script should probably be smarter about this (cc @gsnedders) but for now I'm catching these issues manually.

Reply:
Done.